contributors/devel/sig-testing/e2e-tests.md

```diff
@@ -187,11 +187,15 @@ if any specs are pending.
 --ginkgo.focus="": If set, ginkgo will only run specs that match this regular
 expression.
 
---ginkgo.noColor="n": If set to "y", ginkgo will not use color in the output
-
 --ginkgo.skip="": If set, ginkgo will only run specs that do not match this
 regular expression.
 
+--ginkgo.label-filter="": If set, select tests based on their labels as described under
+"Spec Labels" in https://onsi.github.io/ginkgo/#filtering-specs. This can focus
+on tests and exclude others in a single parameter without using regular expressions.
```

Comment:
it still uses regular expressions no? what it solves is the need to use regex for both focus and skip IIUIRC

Reply:
"without *having* to use regular expressions"?

That's what I wanted to say ("can ... without using regular expressions"), not that it never uses regular expressions.